README.md

```diff
@@ -1,5 +1,46 @@
-# Group13_NamePending
-Group member: Son Chau
-Nikita
-yair
-Aldo
+# Quality white wine predictor
+Authors: Aldo Saltao Barros, Nikita Shimberg, Yair Guterman, Son Chou
+
+## About 
+
+**Goal**
+This project aims to determine which features are the best quality white wine indicators and generate insights into each of these factors to our model’s wine quality.
```

Comment:
I think that last "to our model’s wine quality" can be removed.

Also, is our goal to build a model to predict wine quality given wine features, or is it to determine which features indicate best quality?